src/Resources/skeleton/serializer/Normalizer.tpl.php

```diff
@@ -0,0 +1,30 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+
+class <?= $class_name ?> implements NormalizerInterface
```

Comment:
Shouldn't this be `final`?

Reply:
It could indeed.Voter are not. I was consistant